build/wikibase/callback.sh

```diff
@@ -47,14 +42,34 @@ if [[ "$MW_WG_SERVER" == *.localhost ]]; then
 fi
 
 GRAPHQL_URL="https://wikibase-metadata.toolforge.org/graphql"
+
```

Comment:
I wouldn't report these errors if the user does not have `METADATA_CALLBACK` to true.

Reply:
The script [exits in line 26](https://github.com/wmde/wikibase-release-pipeline/blob/78a45df25a9b20905b0c7180d62fae68b57d59e2/build/wikibase/callback.sh#L26) when callback is not enabled. Is this what you are aiming for. Or what is it, you would like to change?

Reply:
When the user has opted-out of `METADATA_CALLBACK` we will still report errors here that they didn't configure hosts.

Reply:
Hmm. Did you experience that? As I intended, read, and tested the code, the only message I get when opted out is: "METADATA_CALLBACK not enabled." as echo'd in [line 25](https://github.com/wmde/wikibase-release-pipeline/blob/78a45df25a9b20905b0c7180d62fae68b57d59e2/build/wikibase/callback.sh#L25C9-L25C38).

```
❱ cat .env | grep META
METADATA_CALLBACK=false

❱ docker logs wbs-deploy-wikibase-1 | head -n 4
METADATA_CALLBACK not enabled.
/config/LocalSettings.php not found, running MediaWiki install.
PHP 8.3.23 is installed.
ICU 72.1 is installed (supports Unicode 15.0.0).
```